src/main/resources/api/service_openapi.yaml

```diff
@@ -24,6 +24,37 @@ paths:
           description: Service is broken
           $ref: '#/components/responses/StatusResponse'
 
+
+  '/api/janitor/v1/resource/{type}':
+    parameters:
+      - $ref: '#/components/parameters/Type'
+    post:
+      description: |
+        Create a new tracked resource.
+      operationId: createResource
+      tags:
+        - janitor
+      requestBody:
+        description: |
+          Resource UID, time to live, resource labels
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateResourceRequestBody'
+      responses:
+        200:
+          description: Ok
+          $ref: '#/components/responses/CreateResponse'
+        400:
+          description: Bad request - invalid id, badly formed
+          $ref: '#/components/responses/ErrorResponse'
+        401:
```

Comment:
I think 403 is more appropriate. 401 is more like we weren't able to authenticated correctly, 403 for authz.

Reply:
I see. I copied this from WM, they use 401, so I followed their pattern